src/Search/Gearman/Command/QueueCommand.php

```diff
@@ -3,11 +3,11 @@
 namespace eLife\Search\Gearman\Command;
 
 use eLife\ApiClient\Exception\BadResponse;
-use eLife\Search\Monitoring;
-use eLife\Search\Queue\InternalSqsMessage;
```

Comment:
you can probably now remove also some of these classes if they're not used anymore in favor of the ones from `eLife\Bus\`?